orderer/consensus/etcdraft/storage_test.go

```diff
@@ -80,7 +80,7 @@ func TestOpenWAL(t *testing.T) {
 		for i := 0; i < 10; i++ {
 			store.Store(
```

Comment:
I don't understand how this test demonstrates the problem you describe.

Can you reproduce the problem in a unit test?